### DISCUSSION TITLE
Entry clear button color

### DISCUSSION THREAD

**Discussion by Depechie** [xamarin/XamarinCommunityToolkit#1114] [Ideas]
I have the code for an Entry effect with which you can set the Clear button color of the entry on iOS.
Would this be a nice addition to the toolkit?

If so, I can try to take a stab at android, but I guess co-work should be needed.

**Comment by Cfun1** [comment]
@Depechie  I believe it will be helpful, I need to change it as well, till now didn't get time to see how it is done at Android level. You can open a feature-request, I have opened similar one but for cursor color https://github.com/xamarin/XamarinCommunityToolkit/issues/735
